### ISSUE TITLE
`alt` attributes ignored by screen reader on images

### ISSUE THREAD

**Issue by astorije** [hakimel/reveal.js#1854]
When using a screen reader (VoiceOver, in my tests), the whole image is ignored and therefore the SR does not read the `alt` attributes.

As a side note from that issue, is there any quick fix / hack I could apply for a presentation I am giving in a couple days? Thanks!
